src/test/excessiveblock_test.cpp

```diff
@@ -229,6 +238,9 @@ BOOST_AUTO_TEST_CASE(check_generated_block_validator)
 
     str = MiningBlockSizeValidator(tmpMGB, (uint64_t *) 42, true);
     BOOST_CHECK(! str.empty());
+
+    maxGeneratedBlock = c_mgb;
+    excessiveBlockSize = c_ebs;
 }
```

Comment:
Dunno about a1e0357, it seems that even if there's no functional change (enforcing maxGeneratedBlock <= execessiveBlockSize) the commit use the validator rather than an explicit `if`... I'd say that the change could stay in this PR, but I don't have a strong opinion about it